src/HasMedia/MediaCollection.php

```diff
@@ -4,193 +4,211 @@
 
 use Spatie\MediaLibrary\MediaCollection\MediaCollection as ParentMediaCollection;
```

Comment:
@dejwCake if i see correctly, no change in this file, only code style, correct?

@dejwCake @SamuelT-ST But I'm questioning if we still need this file at all. What features would we miss if we just use the Spatie's class?